src/App.js

```diff
@@ -1,25 +1,13 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import List from './pages/List';
```

Comment:
Bit of a nitpick: I'm not sure I get the folder structure here. We have two modules that are in `pages` but they both make up the same page. I would expect a `Home` component to live in pages, and then `AddItem` and `List` would be components.

Reply:
You're totally right. Per the scope of this issue, the folder structure should be much more like you're mentioning. 

I was working ahead and trying to work in the routing code that the other pair wrote but hasn't yet merged. In that PR they are routing to two separate pages, so I set this up so that we could have the List and AddItem components be separate pages. 

I think lets leave this for now since it'll help give direction to the merge conflicts today.

Reply:
Ah ok, that makes sense 👍